airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -310,6 +325,29 @@ else if (hasFailed.get()) {
 
   }
 
+  private class ThreadedTimeHolder {
```

Comment:
One thought - would it be cleaner to have one internal POJO tracking all stat related data here? 

Right now we have `ThreadTimeHolder` and 5 other variables scattered through this class. My sense is it would be more readable if all of these were held by a private POJO - named `GranularSyncTiming` or something along those lines. 

Benefits:
1) Group all stat related info together.
2) Clarify what these are used for.
3) The code block in line 225 - 230 can be replaced with a `toSyncStats` method on the object, that increase readability.

Reply:
Nice! 0eb2eb7e74e476f0479956179bc9108129b3c806